packages/ui/src/components/va-sidebar/VaSidebar.vue

```diff
@@ -7,7 +7,7 @@
     @mouseenter="updateHoverState(true)"
     @mouseleave="updateHoverState(false)"
   >
-    <div class="va-sidebar__menu" :style="`width: ${computedWidth};`">
```

Comment:
Minimized width story is broken (is less than 25%)

<img width="396" alt="image" src="https://github.com/epicmaxco/vuestic-ui/assets/5394573/d384a3b4-9c62-4bb2-ae80-423cd3585dd0">